tests/core/test_simd_float32x4.c

```diff
@@ -5,9 +5,25 @@
 #include <math.h>
 #include <assert.h>
 
+void printFloat(float n) {
```

Comment:
https://github.com/bpowers/musl/commit/0f859fc99325bbb25d506b28565e4a7385ffe0e6

Now it will include the sign for negative nan, and prints `"-nan"`. asmjs and wasm have different ideas of which nans to use when, so this just normalizes them all the time, because the tests don't really care about which nan exactly.